requirements/pip.txt

```diff
@@ -18,42 +18,32 @@ Markdown<3.0
 
 django==1.11.16
 six==1.11.0
-future==0.16.0
-
-# django-tastypie 0.14.0 drops support for django 1.9
-django-tastypie==0.13.3
-
+future==0.17.0
+django-tastypie==0.14.2
 django-guardian==1.4.9
-django-extensions==2.1.2
+django-extensions==2.1.3
 
-# djangorestframework 3.7.x drops support for django 1.9.x
-djangorestframework==3.6.4
+djangorestframework==3.9.0
 
 # Filtering for the REST API
-# django-filter>2.0 require Django 1.11
-# https://django-filter.readthedocs.io/en/master/guide/migration.html#migrating-to-2-0
-# Besides the guide, check this comment for the migration
-# https://github.com/rtfd/readthedocs.org/pull/4318#discussion_r214163531
-django-filter==1.1.0
+django-filter==2.0.0
```

Comment:
If we want to upgrade to `django-filter>=2.0.0` we need to drop support for Python2 since [it requires Python3](https://django-filter.readthedocs.io/en/master/guide/install.html?highlight=python3). Tests fail with:

```
>   from .filterset import FilterSet
E     File "/home/travis/build/rtfd/readthedocs.org/.tox/py27/lib/python2.7/site-packages/django_filters/filterset.py", line 184
E       def __init__(self, data=None, queryset=None, *, request=None, prefix=None):
E                                                     ^
E   SyntaxError: invalid syntax
```